test/jdk/java/foreign/handles/invoker_module/handle/invoker/MethodHandleInvoker.java

```diff
@@ -88,6 +88,8 @@ static <Z> void addDefaultMapping(Class<Z> carrier, Z value) {
         addDefaultMapping(AddressLayout.class, ValueLayout.ADDRESS);
         addDefaultMapping(SymbolLookup.class, SymbolLookup.loaderLookup());
         addDefaultMapping(Consumer.class, (Consumer<Object>)(Object o) -> {});
+        addDefaultMapping(FunctionDescriptor.class, FunctionDescriptor.ofVoid());
+        addDefaultMapping(Linker.Option[].class, null);
```

Comment:
This is a pragmatic change (not too proud of it). Since the linker methods are varargs, invoking these methods with `invokeWithArguments` will expect separate `Linker.Option` parameters instead of a single array. So I pass null instead.